sklearn/covariance/tests/test_covariance.py

```diff
@@ -81,6 +81,20 @@ def test_covariance():
 
 
 def test_shrunk_covariance():
```

Comment:
It could be worth to add a docstring at this test:

```
"""Check the consistency between `ShrunkCovariance` and `shrunk_covariance`."""

Reply:
Done.